thirdparty/embree/common/simd/arm/sse2neon.h

```diff
@@ -5954,7 +5954,13 @@ FORCE_INLINE void _mm_storeu_si32(void *p, __m128i a)
 FORCE_INLINE void _mm_stream_pd(double *p, __m128d a)
 {
 #if __has_builtin(__builtin_nontemporal_store)
+// -- GODOT start --
+#if defined(__aarch64__)
+    __builtin_nontemporal_store(a, (float64x2_t *) p);
+#else
     __builtin_nontemporal_store(a, (float32x4_t *) p);
+#endif
```

Comment:
Would be good to propose this upstream. We're not on the latest version, but the code is still the same there in the latest dev branch:

https://github.com/embree/embree/blob/be0accfd0b246e2b03355b8ee7710a22c1b49240/common/simd/arm/sse2neon.h#L5958-L5967